client/my-sites/plugins/style.scss

```diff
@@ -24,16 +34,6 @@ body.is-section-plugins.theme-default.color-scheme {
 	margin-bottom: 16px;
 }
 
-.upsell-nudge.is-jetpack {
-	margin-top: 51px;
-	margin-bottom: -20px;
-
-	@include breakpoint-deprecated( "<660px" ) {
-		margin-top: 86px;
-		margin-bottom: -50px;
-	}
-}
-
 .upsell-nudge {
```

Comment:
This could end up colliding as well?

Reply:
I believe inline order there won't matter, because the more specific one will always take precedence anyway.

Reply:
Ah yes, you're right.